strongbox-security/strongbox-authentication-providers/strongbox-ldap-authentication-provider/src/test/java/org/carlspring/strongbox/authentication/api/impl/ldap/LdapAuthenticationProviderTest.java

```diff
@@ -43,11 +45,18 @@
 public class LdapAuthenticationProviderTest
 {
 
+    @org.springframework.context.annotation.Configuration
+    @ComponentScan(basePackages = { "org.carlspring.strongbox.configuration"})
+    public static class SpringConfig { }
```

Comment:
it will be better to expectedly add `StrongboxSecurityConfig` to `@ContextHierarchy` of this test class.